core/iwasm/interpreter/wasm_interp_fast.c

```diff
@@ -3536,6 +3541,24 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
                 HANDLE_OP_END();
             }
 
+#if WASM_ENABLE_SIMDE != 0
```

Comment:
That doesn't seem right - it makes the assumption about SIMD implementation; if for some platforms there will be another library used to implement SIMD instructions, you'd have to remember to update this place (and the condition will become more and more complicated). We should attempt to refactor the code so the simple and universal condition `WASM_ENABLE SIMD != 0` should be sufficient.

Reply:
let's tackle this separately and out of this PR